src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -98,6 +98,23 @@ public <T> Future<Void> put(String endpoint, T dataObject, RequestContext reques
       .mapEmpty();
   }
 
+  public <T> Future<T> put(String endpoint, T dataObject, Class<T> responseType ,RequestContext requestContext) {
+    log.info(REQUEST_MESSAGE_LOG_INFO, HttpMethod.PUT, endpoint);
+
+    var recordData = JsonObject.mapFrom(dataObject);
+    log.debug(REQUEST_MESSAGE_LOG_DEBUG, () -> HttpMethod.PUT, () -> JsonObject.mapFrom(recordData).encodePrettily());
```

Comment:
can provide sensitive info in the logs, we could not control here what will be logged, basically everething from finance information can be logged

Reply:
removed it, thanks